src/mesh/accessors_impl.hh

```diff
@@ -225,7 +224,7 @@ inline bool Side::is_boundary() const {
 }
 
 inline NodeAccessor<3> Side::node(unsigned int i) const {
```

Comment:
Do we need `side_nodes` data structure and global numbering of sides? That was necessary for MH, but it is now replaced by universal dofhandler. We can possibly use element_idx + local side number in the Side accessor, then
the element accessor can be used to obtain nodes.